heartbeat/scheduler/scheduler.go

```diff
@@ -229,15 +221,16 @@ func (s *Scheduler) Add(sched Schedule, id string, entrypoint TaskFunc, jobType
 	}, nil
 }
 
-func (s *Scheduler) runOnce(runAt time.Time, taskFn timerqueue.TimerTaskFn) {
+// runTaskOnce runs the given task exactly once at the given time. If
+func (s *Scheduler) runTaskOnce(runAt time.Time, taskFn timerqueue.TimerTaskFn, deadlineCheck bool) {
 	now := time.Now().In(s.location)
-	if runAt.Before(now) {
-		// Our last invocation went long!
+	// Check if the task is more than 1 second late
```

Comment:
I think the previous check was too sensitive, basically asking to run a task at an exact time and complaining if it ran even 1ns late. Weird that this wasn't an issue previously. At any rate 1s is more reasonable